src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -90,6 +118,11 @@ class ProfileCallTreeContextMenu extends PureComponent {
           >
             Function Name
           </MenuItem>
+          {this.isJS()
```

Comment:
Yeah, I made it a function just for readability and not leaking out unneeded variables. I'm fine with changing it. Do you want me to inline it?